### PR TITLE
Add landscape payment article

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: content/articles/looking-at-payments.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Short_summary: Payments in Europe are complicated due to VAT, I look at what a SAAS could use
+Category: Product
+Authors: Vincent
+
+
+As we are working on our product (if you work in an agency, we'd love to have a chat with you! send us a mail at team AT wearewizards.io), I was curious to see the payments landscape in Europe, having read so much about [#VATMESS](https://twitter.com/hashtag/vatmess).
+<!-- PELICAN_END_SUMMARY -->
+
+Most Software As A Service business operate with monthly plans with various tiers. 
+In Europe, before 2015, the supplier would charge their own VAT rate to their customers: for example if the supplier was based in the UK and the customer in France, the supplier would charge the UK VAT rate.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: "would charge the VAT rate of their own country" maybe?

File: content/articles/looking-at-payments.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+since 2015 however, the supplier now has to charge the VAT rate where the customer is located and report sales to that country. This change was made to prevent unfair competitions from countries with a lower VAT rate, with big companies moving to Luxembourg for example. Regulators only forgot to set a lower limit (they are apparently now thinking of a £5k limit) and didn't think of SME or thought about it and decided that crippling european businesses would be a good idea.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: "the VAT rate of the customer's country" ?

Comment (teh) [inline content/articles/looking-at-payments.md]: "unfair competition"

File: content/articles/looking-at-payments.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+This article will focus on what it means for a UK based VAT registered company as it is our case.
+
+
+# Business side
+The UK has put in place something called the VAT Mini One Stop Shop or VAT MOSS (https://www.gov.uk/guidance/register-and-use-the-vat-mini-one-stop-shop). This allows businesses to register in only one country and let it redistribute the money after filing a VAT return for other countries on a quarterly basis.
+
+If the customer is a business and is VAT registered, just getting the VAT number and checking it should be enough. However, companies paying without giving their VAT number or non-VAT registered ones should be treated as B2C.
+
+As mentioned in the introduction, we need to charge the VAT of the customer which means we need to locate it.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: * "to charge the VAT of the customer's country"
* "locate it -> locate them"

File: content/articles/looking-at-payments.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+Here are some examples of pieces of informations we can use:
```
Comment (teh) [inline content/articles/looking-at-payments.md]: Here are some examples for information we can use to locate a customer:

File: content/articles/looking-at-payments.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+- billing address
+- ip address geolocation
```
Comment (teh) [inline content/articles/looking-at-payments.md]: IP address

File: content/articles/looking-at-payments.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+- bank location
+- card location
+
+Note that the legislation requires to have 2 non-conflicting informations and keep that for 10 (!) years.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: two non-conflicting items of information and keep them for 10

File: content/articles/looking-at-payments.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+Here are some examples of pieces of informations we can use:
+
+- billing address
+- ip address geolocation
+- bank location
+- card location
+
+Note that the legislation requires to have 2 non-conflicting informations and keep that for 10 (!) years. 
+If you don't manage that, you are supposed to reject the transaction. You could possibly ask for the user to send some proof by mail but that's a terrible user experience.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: Without the non-conflicting information you have to reject the transaction.

File: content/articles/looking-at-payments.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+If we manage to get customers to pay us, we then need to separate sales by country and apply the right VAT. This gives us the amount of tax we need to pay for each country.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: Not sure what this sentence means after you talked about vatmoss above?

File: content/articles/looking-at-payments.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+# Coding
+Now that we have an overview of what the VAT system looks like, let's have a look at the solutions.
+
+## Payment providers
+None of these actually help with the VAT mess but are still needed.
+There are a few alternatives here:
+
+- [Stripe](https://stripe.com)
+- [Braintree](https://www.braintreepayments.com/)
+- [Paymill](https://www.paymill.com/)
```
Comment (teh) [inline content/articles/looking-at-payments.md]: I linked to a provider that did vat moss a while back, can you remember which one that was?

Comment (teh) [inline content/articles/looking-at-payments.md]: Ah, nevermind you talk about it below.

Comment (teh) [inline content/articles/looking-at-payments.md]: A few more: http://fastspring.com/ , https://www.chargebee.com/a/

File: content/articles/looking-at-payments.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+Paymill is based in Europe. Stripe is extremely easy to setup and get started. I haven't used the other two so I can't comment on those but it seems you need to be approved for a merchant account for them which means some some wait.
+
+Here are the advertised rates (you can possibly get a better one by talking with them):
```
Comment (teh) [inline content/articles/looking-at-payments.md]: "you can" -> "you might" or "you could"

File: content/articles/looking-at-payments.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+- Paymill: 2.95% + 0.28€
+- Braintree: 2.4% + 20p (first £30k free) for EU cards, 3.4% + 20p for non EU cards
+- Stripe: 1.9% + 20p for UK cards and 2.9% + 20p for foreign cards
+
+Let's use a small example to see the difference: £10k sales with £5k from the UK, £3k from NA and £2k from Europe with let's say 100 transactions
```
Comment (teh) [inline content/articles/looking-at-payments.md]: maybe end in `:` ?

File: content/articles/looking-at-payments.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+Stripe seems to give the best price but keep in mind you cand probably get better rates than the advertised one (at least for Paymill according to the employee I met ages ago).
+
+## Handling VAT
+
+### Implementing it yourself
+You need to handle a few cases:
+
+#### Customer is VAT registered
+Ask for VAT number and use your country VAT is customer is based in your country or no VAT otherwise
```
Comment (teh) [inline content/articles/looking-at-payments.md]: your countries' VAT if the customer ...

Comment (teh) [inline content/articles/looking-at-payments.md]: Also don't understand the end of that sentence

File: content/articles/looking-at-payments.md [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+
+#### Customer doesn't provide VAT number
+Ask for country (that can be prefilled based on IP address) and compare with the IP location. If it matches continue otherwise compare the country filled with the bank country. If it matches continue otherwise ask the customer to send a fax with their certificate of birth and a passport. More seriously what should be done in the last case?
```
Comment (teh) [inline content/articles/looking-at-payments.md]: * Ask for "a" country
* the country provided (not filled)

More seriously: what  ...

File: content/articles/looking-at-payments.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+
+- [Recurly](https://recurly.com/): $99/month and 10¢ per transaction + 1.25% of revenue (on top of payment gateway)
+- [Quaderno](https://quaderno.io/): $29/month on top of your payment provider, provides a widget for checkout that handles European VAT rules
+- [Taxamo](https://www.taxamo.com): £40/months for all the regions, pretty limited number of country as it's missing Canada and Australia for example. Doesn't actually show what the product is but included for completeness
+
+Out of those, Quaderno seems to be the best choice and is fairly cheap.
+
+
+## Conclusion
+As a developer I would be tempted to write my own version but Quaderno seems like a good fit and is cheap enough for a B2B SAAS that it might be worth avoiding the hassle of coding it.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: "I am tempted" maybe?

File: content/articles/looking-at-payments.md (hunk 8 above)
Comment (Keats) [inline content/articles/looking-at-payments.md]: i'm thinking of putting the VATMOSS below that, will think about it overnight

File: content/articles/looking-at-payments.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
+Title: Looking at payments solution for SAAS in Europe
+Date: 2015-12-08
+Short_summary: Payments in Europe are complicated due to VAT, I look at what a SAAS could use
```
Comment (teh) [inline content/articles/looking-at-payments.md]: summary is not a full sentence.

"Payments in Europe are complicated due to recent VAT changes. I researched several SAAS that handle the new rules." maybe?

File: content/articles/looking-at-payments.md [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+Authors: Vincent
+
+
+As we are working on our product (if you work in an agency, we'd love to have a chat with you! send us a mail at team AT wearewizards.io), I was curious to see the payments landscape in Europe, having read so much about [#VATMESS](https://twitter.com/hashtag/vatmess).
+<!-- PELICAN_END_SUMMARY -->
+
+Most Software As A Service business operate with monthly plans with various tiers. 
+In Europe, before 2015, the supplier would charge the VAT rate of their own country to their customers: for example if the supplier was based in the UK and the customer in France, the supplier would charge the UK VAT rate.
+
+Since 2015 however, the supplier now has to charge the VAT rate of the customer's country and report sales to that country. This change was made to prevent unfair competition from countries with a lower VAT rate, with big companies moving to Luxembourg for example. Regulators only forgot to set a lower limit (they are apparently now thinking of a £5k limit) and didn't think of SME or thought about it and decided that crippling european businesses would be a good idea.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: european -> European

File: content/articles/looking-at-payments.md [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+- IP address geolocation
+- bank location
+- card location
+
+Note that the legislation requires to have two non-conflicting items of information and keep them for 10 (!) years. 
+Without the non-conflicting information you have to reject the transaction. You could possibly ask for the user to send some proof by mail but that's a terrible user experience.
+
+If we manage to get customers to pay us, we then need to separate sales by country and apply the right VAT. This gives us the amount of tax we need to pay for each country.
+
+The UK has put in place something called the VAT Mini One Stop Shop or VAT MOSS (https://www.gov.uk/guidance/register-and-use-the-vat-mini-one-stop-shop). This allows businesses to register in only one country and let it redistribute the money after filing a VAT return for other countries on a quarterly basis with the amounts we calculated.
```
Comment (teh) [inline content/articles/looking-at-payments.md]: I think the limit for VATMOSS is 81k as well, maybe worth mentioning?

File: content/articles/looking-at-payments.md [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+Let's use a small example to see the difference: £10k sales with £5k from the UK, £3k from NA and £2k from Europe with let's say 100 transactions:
+
+```python
+paymill = (2.95 * 10000 / 100) + (0.20 * 100) # assuming £1=1€
+>> 315
+braintree = (2.4 * 7000 / 100) + (3.4 * 3000 / 100) + (0.20 * 100)
+>> 290
+stripe = (1.9 * 5000 / 100) + (2.9 * 5000 / 100) + (0.20 * 100)
+>> 260
+fastspring = (5.95 * 10000 / 100) + (0.63 * 100) # today's rate $0.95 > £0.63
```
Comment (teh) [inline content/articles/looking-at-payments.md]: fastspring doesn't compete with stripe, it competes with recurly etc but interesting nevertheless

Comment (Keats) [inline content/articles/looking-at-payments.md]: afaik fastspring do compete with stripe, it's a payment provider?

File: content/articles/looking-at-payments.md (hunk 17 above)
Comment (Keats) [inline content/articles/looking-at-payments.md]: afaik with VATMOSS there is no threshold?

**Comment by teh** [comment]
LGTM

**Pull request merged**